services/mining/newblocktemplate.go

```diff
@@ -278,6 +278,13 @@ mempoolLoop:
 
 	blockSize := uint32(blockHeaderOverhead) + uint32(coinbaseTx.Transaction().SerializeSize()) + tokenSize
 
+	// ==== fix parents size
+	blockSize += uint32(s.VarIntSerializeSize(uint64(len(parents))))
+	for i := 0; i < len(parents); i++ {
```

Comment:
parents is nil

Reply:
if parents == nil {
		parents = blockManager.GetChain().GetMiningTips(blockdag.MaxPriority)
	}